main/realmain.cc

```diff
@@ -240,6 +245,31 @@ void runAutogen(core::Context ctx, options::Options &opts, WorkerPool &workers,
         auto last = unique(mergedClasslist.begin(), mergedClasslist.end());
         opts.print.AutogenClasslist.fmt("{}\n", fmt::join(mergedClasslist.begin(), last, "\n"));
     }
+    if (opts.print.AutogenSubclasses.enabled) {
+        Timer timeit(logger, "autogenSubclassesPrint");
+
+        // Merge the {Parent: Set{Child1, Child2}} maps from each thread
+        map<string, set<string>> mergedSubclasses;
+        for (auto &el : merged) {
+            for (auto const &[parentName, children] : el.second.subclasses) {
+                if (parentName.empty()) {
+                    // Child < NonexistentParent
+                    continue;
+                }
+                mergedSubclasses[parentName].insert(children.begin(), children.end());
+            }
+        }
+
+        vector<string> mergedSubclasseslist;
+        for (auto const &[parentName, children] : mergedSubclasses) {
```

Comment:
This is a lot of string copying. Trying to think of if we can use refs more effectively here....